shell/pom.xml

```diff
@@ -25,6 +25,11 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.reflections</groupId>
+      <artifactId>reflections</artifactId>
+      <version>0.9.10</version>
+    </dependency>
```

Comment:
What's the benefit of using reflection to populate the commands list as opposed to enumerating them? It doesn't seem like there will be too many commands.